tests/suites/test_suite_ccm.function

```diff
@@ -457,3 +454,114 @@ exit:
     mbedtls_free( io_msg_buf );
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void mbedtls_ccm_skip_auth( int cipher_id, int mode,
```

Comment:
“skip auth” makes me think this is about the tag. I think this should be `mbedtls_ccm_skip_ad`.